cldf/languages.csv

```diff
@@ -1,23 +1,23 @@
-ID,Name,Glottocode,Glottolog_Name,ISO639P3code,Macroarea,Latitude,Longitude,Family
-bichom,bichom,bich1234,Bichom,,,,,
-bulu,bulu,bulu1255,Bulu,,,,,
-chayangtajo,chayangtajo,bugu1245,Chayangtajo,,,,,
-dikhyang,dikhyang,dikh1234,Dikhyang,,,,,
-duhumbi,duhumbi,chug1252,Chug,,,,,
-jerigaon,jerigaon,jeri1243,Jerigaon,,,,,
-kaspi,kaspi,kasp1234,Kaspi,,,,,
-khispi,khispi,lish1235,Lish,,,,,
-khoina,khoina,khoi1253,Khoina,,,,,
-khoitam,khoitam,khoi1252,Khoitam,,,,,
-kojorojo,kojorojo,kojo1234,Kojo Rojo,,,,,
-lasumpatte,lasumpatte,lasu1234,Lasumpatte,,,,,
-lidaqin,lidaqin,sulu1241,Eastern Puroik,,,,,
-namphri,namphri,namp1239,Namphri,,,,,
-rahung,rahung,rahu1234,Rahung,,,,,
-rawa,rawa,rawa1269,Rawa (Puroik),,,,,
-rupa,rupa,rupa1234,Rupa,,,,,
-saria,saria,sari1249,Sario Saria,,,,,
-shergaon,shergaon,sher1261,Shergaon,,,,,
-singchung,singchung,sing1271,Singchung,,,,,
-wangho,wangho,wang1301,Wangho,,,,,
-zmyyc,zmyyc,sulu1241,Eastern Puroik,,,,,
+ID,Name,Glottocode,Glottolog_Name,ISO639P3code,Macroarea,Latitude,Longitude,Family,Source
```

Comment:
Ah, sure, I'll do it right now.

Reply:
Good thing you asked for that, because there was a serious problem: two languages with `lidaqin` as ID! And the `Name` column needs improvements as well, I will check the source to make sure they were not mixed.

Reply:
Thanks, just merge once this is done. And maybe also add a version, so we can already publish this with Zenodo next week? When doing so, make sure to git-checkout the versions 2.1 for concepticon and 4.0 for glottolog, so we don't have the version hash statements, but a plain version for concepticon and glottolog.

Reply:
>  two languages with lidaqin as ID! 
When working with all the other SEA datasets recently, I found many small things which I overlooked rather coincidentally. There's some reason why other people have not done this in this degree before. This should be added to our checklist of lexibank guidelines: does the dataset have as many languages as claimed?

Reply:
This was indeed problematic: I was wondering why `cldf validate` didn't catch it, and it is because, when creating the CLDF, all the raw forms under the `lidaqin` languages were being added to the first entry listed in `etc/languages.csv`...